Quadriga/src/main/webapp/WEB-INF/tiles-defs.xml

```diff
@@ -829,8 +829,22 @@
 		<put-attribute name="content"
 			value="/WEB-INF/views/editing/networkshistory.jsp" />
 	</definition>
-	
-	
+
+	<definition name="base.definition.public"
```

Comment:
This leads to the other public pages not having a navigation anymore:
![image](https://cloud.githubusercontent.com/assets/8881141/13999248/93efd256-f0f8-11e5-8add-10fce8ed2986.png)